VehicleIdentificationMsg identifiers

VehicleTypeMsg type
float32 state_of_charge
string preferred_charge_service_provider

time preferred_charge_time_begin
time preferred_charge_time_end
